sim/core/target.go

```diff
@@ -228,8 +227,9 @@ type AttackTable struct {
 	PartialResistShadowThresholds Thresholds
 
 	DamageDealtMultiplier               float64
-	NatureDamageDealtMultiplier         float64
-	PeriodicShadowDamageDealtMultiplier float64
+	DamageTakenMultiplier               float64
```

Comment:
Having `XXXTakenMultiplier` (as opposed to `XXXDealtMultiplier`) doesn't make sense on an `AttackTable`, because the table is inherently mono-directional, i.e. is only used with a particular attacker and particular defender.

Reply:
The XXXDealtMultipliers are all used in applyTargetModifiers(), so they were acting like XXXTakenMultipliers to start with. I think they were introduced for Black Arrow and Noxious Sting, which both act like "Mod Damage Taken" multipliers (they apply to ticking DoTs), and have then been extended to Crusade or Improved Tracking: those, however, are "Mod Damage Done" effects, only applying to DoT snapshotting.

Reply:
Ahh ok, that makes sense then, thank you.

Reply:
Yes, I'll add this in the next PR.

Reply:
This isn't the type of thing to do in the next PR; I'm just asking for a comment.